travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - "node"
+
```

Comment:
can you add the install and script that runs like below?

install:
  - npm install

script:
  - npm test
  - npm run coverage

OR 

install:
  - npm install

script:
  - npm test

after_success:
 - npm run coverage

Reply:
I just added the install and script commands.. doesn't seems to change anything.
still trying to figure out whats wrong.

Reply:
I fixed the issue @dinorhythms.. thanks